integration/terraform/ec2/linux/README.md

```diff
@@ -19,3 +19,64 @@ AMI builder pipeline builds the ami
 The pipeline installs required packages and updates ami software
 
 This process generates a new ami we can then use for testing
+
+**How to integration test in your aws account**
+1. Create resources and setup local
+   1. Install terraform
+   2. Set up aws credentials
+   3. Set up iam role
+      1. Role must include cloud watch server policy and s3 policy
+   4. Create s3 bucket
+2. Upload binary to s3
+   1. make release 
+      1. hints
+         1. You may want to do this on a linux ec2 instance installing the agent may fail if you build on Mac 
+         2. If you want to build faster remove not needed packages and test from release step
+   2. aws s3 cp build/bin s3://${your bucket name}/integration-test/binary/${git commit sha} --recursive
+3. Start Local Stack
+   1. Go to Local Stack directory
+      1. cd ../localstack
+   2. init terraform
+      1. terraform init
+   3. Apply terraform 
+      1. ```
+         terraform apply --auto-approve \
+         -var="github_repo=${gh repo you want to use ex https://github.com/aws/amazon-cloudwatch-agent.git}" \
+         -var="github_sha=${commit sha you want to use ex fb9229b9eaabb42461a4c049d235567f9c0439f8}" \
+         -var='vpc_security_group_ids=["${name of your security group}"]' \
+         -var="key_name=${name of key pair your created}" \
+         -var="s3_bucket=${name of your s3 bucket created}" \
+         -var="iam_instance_profile=${name of your iam role created}" \
+         -var="ssh_key=${your key that you downloaded}"
+         ```
```

Comment:
You could add snippets of the expected output, hopefully just a few important lines, not 100s of lines.